pages/common/from.md

```diff
@@ -0,0 +1,20 @@
+# from
+
+> Prints mail header lines from your mailbox.
```

Comment:
@sbrl Should we use `email` instead of `mail`?

Reply:
Hrm, good question. The man page tends to favour `mail` over email, so I'd suggest doing the same on this page.